src/averageAges.js

```diff
@@ -15,17 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filtered = century
+    ? people.filter(a => a.sex === 'm')
+      .filter(a => Math.ceil(a.died / 100) === century)
+    : people.filter(a => a.sex === 'm');
```

Comment:
```suggestion
  const filtered = people.filter(a => century 
    ? a.sex === 'm' && Math.ceil(a.died / 100) === century
    : a.sex === 'm'
  );
```
Выражение можно упростить зачем дважды фильтровать, если можно один раз?
Плюс `a` плохое название переменной, по названию должно быть понятно что внутри(там объект с информацией о человеке)